Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/AssetTests.cs

```diff
@@ -138,216 +189,453 @@ public async Task CreateAssetAsync_CreatesDynamicAssetModel()
         }
         
         [Fact]
-        public async Task CreateAssetAsync_Null_DynamicModel_Throws()
+        public async Task CreateAssetAsync_DynamicallyTyped_IdentifierIsNull_Throws()
         {
-            var client = _fileSystemFixture.CreateMockClientWithResponse("Asset.json");
+            var client = _fileSystemFixture.CreateMockClientWithoutResponse();
 
             await client.Invoking(c => c.CreateAssetAsync(null))
                 .Should().ThrowExactlyAsync<ArgumentNullException>();
         }
-        
+
         [Fact]
-        public async Task CreateAssetAsync_CreatesStronglyTypedAssetModel()
+        public async Task CreateAssetAsync_StronglyTyped_WithFileContent_CreatesAsset()
         {
-            var client = _fileSystemFixture.CreateMockClientWithResponse("Asset.json");
-        
+            var client = _fileSystemFixture.CreateMockClientWithResponse("File.json", "Asset.json");
+
             var expected = GetExpectedStronglyTypedAssetModel();
-        
-            var createModel = new AssetCreateModel<ComplexTestModel>
+
+            MemoryStream stream = new MemoryStream(Encoding.UTF8.GetBytes("Hello world from CM API .NET SDK"));
+            string fileName = "Hello.txt";
+            string contentType = "text/plain";
```

Comment:
few other places aswell

```suggestion
            var stream = new MemoryStream(Encoding.UTF8.GetBytes("Hello world from CM API .NET SDK"));
            var fileName = "Hello.txt";
            var contentType = "text/plain";
```

Reply:
Fixed